README.md

```diff
@@ -55,16 +50,18 @@ TODO: discuss CALM Theorem and its connection with Datalog
 
 # 2. Design
 
+TODO: mention open world assumption
+
 ## 2.1 Types
 
-PomoDB is designed to be run within WebAssembly, and so its types and their encodings are informed by the format. See the [WebAssembly specification](https://webassembly.github.io/spec/core/appendix/index-types.html) for more information.
+PomoDB is designed to be run within WebAssembly, and so its types and their encodings are informed by the format. See the [WebAssembly specification][Wasm Types] for more information.
```

Comment:
> designed to be run within WebAssembly, and so its types and their encodings are informed by the format

This is still a choice though, right? Unlike IPVM, we could write _whatever_ types and lower them into Wasm, or am I missing something? It's just an efficiency choice, ya?